system/box64.box64rc

```diff
@@ -198,6 +198,11 @@ BOX64_MALLOC_HACK=2
 BOX64_NOSANDBOX=1
 BOX64_INPROCESSGPU=1
 
+[qq]
+BOX64_MALLOC_HACK=2
+BOX64_NOSANDBOX=1
+BOX64_INPROCESSGPU=1
+
```

Comment:
4 32bit-only opcodes in DynaRec and a 64bit Linux app in the rcfile? Are these changes irrelevant? otherwise it’s confusing.

Reply:
Ok, then I'll delete the pr, qq this can be added